docs/highlights/2020-10-27-metrics-sources.md

```diff
@@ -0,0 +1,62 @@
+---
+last_modified_on: "2020-10-27"
+$schema: ".schema.json"
+title: "New `*_metrics` sources"
+description: "A foray into collecting metrics."
+author_github: "https://github.com/binarylogic"
+pr_numbers: [1314]
+release: "0.11.0"https://github.com/Lusitaniae/apache_exporter
+hide_on_release_notes: false
+tags: ["type: announcement"]
+---
+
+Vector 0.11 includes two new metrics sources:
+
+1. The [`apache_metrics` source][apache_metrics_source]
```

Comment:
Host metrics?

Reply:
I don't think we're stabilizing internal metrics yet, are we?

Reply:
Yes, we are.